charts/weave-gitops/wego-config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+data:
+  config: |
```

Comment:
Do we need this configmap anymore?  Can you confirm whether it was removed from our codebase.

Reply:
The upgrade command still uses it. All the others dont anymore. We may want the map in the future. If we dont we can toss it later.